data/src/main/java/com/architect/coders/mu8/data/events/EventsMapper.kt

```diff
@@ -0,0 +1,15 @@
+package com.architect.coders.mu8.data.events
+
+import com.architect.coders.mu8.data.mapper.BaseResponseMapper
+import com.architect.coders.mu8.data.utils.replaceHttps
+import com.architect.codes.mu8.events.Event
+
+class EventsMapper : BaseResponseMapper<EventsResponse, Event> () {
+    override fun transform(input: EventsResponse): Event {
```

Comment:
Put a new line above this to improve reading.

Reply:
Done